.travis.yml

```diff
@@ -43,7 +44,7 @@ env:
   - TARGET=SPRACINGF3
   - TARGET=SPRACINGF3EVO
 #  - TARGET=SPRACINGF3MINI
-  - TARGET=STM32F3DISCOVERY
```

Comment:
Yeah betaflightf3 is yet just a fictive board. Its a concept what may be
changed.

The code is for now just exposed to do an easy share with hardware
manufactorer